samples/subsys/usb/mass/boards/rak5010_nrf52840.conf

```diff
@@ -0,0 +1,23 @@
+# Storage is on MX25R64
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+
+# Must match the registered name for the selected disk access variant:
+# "RAM" via DISK_RAM_VOLUME_NAME for DISK_ACCESS_RAM
+# "NAND" via DISK_FLASH_VOLUME_NAME for DISK_ACCESS_FLASH
+CONFIG_MASS_STORAGE_DISK_NAME="NAND"
+
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+CONFIG_FLASH_LOG_LEVEL_INF=y
+
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_DISK_ACCESS_FLASH=y
+CONFIG_DISK_FLASH_DEV_NAME="IS25WP064A"
```

Comment:
The difference to `samples/subsys/usb/mass/boards/nrf52840dk_nrf52840.conf ` is just CONFIG_DISK_FLASH_DEV_NAME. We should not have an conf-, overlay-file set for every nRF52xxx based board / |flash combination, I see no benefit for the project, only maintenance overhead.

Reply:
Ok indeed I will change this :)